docs/api/server/Environment-Variables.md

```diff
@@ -714,6 +725,86 @@ ONE_SERVICE_WORKER=true
 ONE_SERVICE_WORKER=false
 ```
 
+## `OTEL_EXPORTER_OTLP_LOGS_ENDPOINT`
+
+**Runs In**
+* ✅ Production
+* ✅ Development
+
+When set, One App will emit OpenTelemetry logs over GRPC to the configured endpoint.
```

Comment:
i think the g is lowercase
```suggestion
When set, One App will emit OpenTelemetry logs over gRPC to the configured endpoint.
```

Reply:
fixed in 29f0b97550d0adf1d23e2f68015c19bcf50f5dd1